adhocteam/Head-Start-TTADP
# Remove total grants statistics from the TTA overview widget

## Description of change
Removes the total grants statistics from the display on the TTA Overview widget. The data retrieval was not removed, since the plans are to bring the data back soon.


## How to test
- pull changes
- verify that (of x) is not showing under grants on the TTA overview widget.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-143
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-243


## Checklists

### Every PR


- [ n/a] API Documentation updated
- [ n/a] Boundary diagram updated
- [ n/a] Logical Data Model updated
- [ n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
